<div style="background:linear-gradient(135deg,#042f2e 0%,#0f766e 55%,#14b8a6 100%);border-radius:18px;padding:34px 30px;color:#fff;font-family:Inter,Segoe UI,sans-serif">
  <div style="font-size:12px;letter-spacing:3px;color:#99f6e4;font-weight:700;text-transform:uppercase">Chapter 40 · Solutions</div>
  <div style="font-size:34px;font-weight:900;line-height:1.1;margin:10px 0 6px">Sampling &amp; the CLT, Worked Answers ✅</div>
  <div style="font-size:15px;color:#d1fae5;max-width:720px;line-height:1.6">Full solutions to the five challenges. Try them yourself first, then compare.</div>
  <div style="margin-top:16px;font-size:13px;color:#a7f3d0">Statistics, Data Science and AI: A Visual Handbook · John Fisher · 2026</div>
</div>

## ⚙️ Setup

In [1]:
import numpy as np
from scipy import stats
rng = np.random.default_rng(401)
print("ready")

ready


<div style="background:#ecfdf5;border-left:5px solid #0d9488;border-radius:10px;padding:16px 20px;font-family:Inter,sans-serif">
<span style="font-size:13px;font-weight:800;color:#0d9488;letter-spacing:1px">CHALLENGE 1 · STANDARD ERROR</span>
<div style="font-size:21px;font-weight:800;color:#1a2138;margin-top:4px">sigma over root n</div>
<div style="color:#4a5578;margin-top:6px">A population has standard deviation 20. Find the standard error of the sample mean for n = 100 and for n = 400.</div>
</div>

In [2]:
sigma = 20
for n in [100, 400]:
    print(f"n={n}: SE = 20/sqrt({n}) = {sigma/np.sqrt(n):.2f}")

n=100: SE = 20/sqrt(100) = 2.00
n=400: SE = 20/sqrt(400) = 1.00


**Answer:** SE = &#963;/&#8730;n, so for n = 100, SE = 20/10 = **2.0**, and for n = 400, SE = 20/20 = **1.0**. Quadrupling the sample halved the standard error, the square-root law in action.

<div style="background:#ecfdf5;border-left:5px solid #0d9488;border-radius:10px;padding:16px 20px;font-family:Inter,sans-serif">
<span style="font-size:13px;font-weight:800;color:#0891b2;letter-spacing:1px">CHALLENGE 2 · CLT IN ACTION</span>
<div style="font-size:21px;font-weight:800;color:#1a2138;margin-top:4px">A probability about a mean</div>
<div style="color:#4a5578;margin-top:6px">Daily sales have mean 500 and sd 120 (skewed). For a sample of n = 36 days, find P(sample mean > 540), using the CLT.</div>
</div>

In [3]:
mu, sd, n = 500, 120, 36
SE = sd/np.sqrt(n)
z = (540 - mu)/SE
print(f"SE = {SE:.2f}, z = (540-500)/SE = {z:.2f}")
print(f"P(sample mean > 540) = {1 - stats.norm.cdf(z):.4f}")

SE = 20.00, z = (540-500)/SE = 2.00
P(sample mean > 540) = 0.0228


**Answer:** the CLT makes the sample mean approximately normal with SE = 120/6 = 20. Then z = (540 &minus; 500)/20 = 2.0, so P(mean &gt; 540) = 1 &minus; &Phi;(2) &approx; **0.0228**. Even though daily sales are skewed, the *mean* of 36 days is nearly normal.

<div style="background:#ecfdf5;border-left:5px solid #0d9488;border-radius:10px;padding:16px 20px;font-family:Inter,sans-serif">
<span style="font-size:13px;font-weight:800;color:#0d9488;letter-spacing:1px">CHALLENGE 3 · THE sqrt(N) LAW</span>
<div style="font-size:21px;font-weight:800;color:#1a2138;margin-top:4px">How much more data?</div>
<div style="color:#4a5578;margin-top:6px">Your current sample gives a standard error of 4. You want to cut it to 1. By what factor must you increase the sample size?</div>
</div>

In [4]:
# SE scales like 1/sqrt(n); to divide SE by 4, multiply n by 4^2
factor = (4/1)**2
print(f"to reduce SE by a factor of 4, increase n by {factor:.0f}x")

to reduce SE by a factor of 4, increase n by 16x


**Answer:** since SE &prop; 1/&#8730;n, dividing the SE by 4 requires multiplying n by 4&sup2; = **16**. Precision improves only with the square root of sample size, so each extra digit of accuracy is dramatically more expensive.

<div style="background:#ecfdf5;border-left:5px solid #0d9488;border-radius:10px;padding:16px 20px;font-family:Inter,sans-serif">
<span style="font-size:13px;font-weight:800;color:#db2777;letter-spacing:1px">CHALLENGE 4 · CONFIDENCE INTERVAL</span>
<div style="font-size:21px;font-weight:800;color:#1a2138;margin-top:4px">Mean plus or minus a margin</div>
<div style="color:#4a5578;margin-top:6px">A sample of 64 has mean 50 and the population sd is 16. Build an approximate 95% confidence interval for the true mean.</div>
</div>

In [5]:
xbar, sigma, n = 50, 16, 64
SE = sigma/np.sqrt(n)
lo, hi = xbar - 1.96*SE, xbar + 1.96*SE
print(f"SE = 16/8 = {SE:.1f}")
print(f"95% CI = 50 +/- 1.96*{SE:.1f} = [{lo:.2f}, {hi:.2f}]")

SE = 16/8 = 2.0
95% CI = 50 +/- 1.96*2.0 = [46.08, 53.92]


**Answer:** SE = 16/&#8730;64 = 2.0, so the 95% interval is 50 &plusmn; 1.96(2) = **[46.08, 53.92]**. We are about 95% confident the true mean lies in this range, an interval built directly on the CLT.

<div style="background:#ecfdf5;border-left:5px solid #0d9488;border-radius:10px;padding:16px 20px;font-family:Inter,sans-serif">
<span style="font-size:13px;font-weight:800;color:#0f766e;letter-spacing:1px">CHALLENGE 5 · A POLL'S MARGIN</span>
<div style="font-size:21px;font-weight:800;color:#1a2138;margin-top:4px">Standard error of a proportion</div>
<div style="color:#4a5578;margin-top:6px">A poll of 1,000 people finds 60% support. Find the standard error of the proportion and the 95% margin of error.</div>
</div>

In [6]:
p, n = 0.60, 1000
SE = np.sqrt(p*(1-p)/n)
print(f"SE = sqrt(0.6*0.4/1000) = {SE:.4f}")
print(f"95% margin of error = 1.96*SE = +/- {1.96*SE:.4f}  (about +/- 3 points)")

SE = sqrt(0.6*0.4/1000) = 0.0155
95% margin of error = 1.96*SE = +/- 0.0304  (about +/- 3 points)


**Answer:** SE = &#8730;(0.6&middot;0.4/1000) &approx; **0.0155**, so the 95% margin of error is 1.96 &times; 0.0155 &approx; **&plusmn;0.030**, the familiar "&plusmn;3 percentage points" of political polls. A proportion is just a mean of 0s and 1s, so the same CLT machinery applies.

---
<div style="background:#ffffff;border:1px solid #e6e9f2;border-radius:16px;padding:22px 26px;font-family:Inter,sans-serif">
<div style="font-size:19px;font-weight:800;color:#1a2138">🎉 Nicely done</div>
<div style="color:#4a5578;line-height:1.8;margin-top:8px">You computed standard errors, used the CLT to find a probability about a mean, applied the square-root law, built a confidence interval, and found a poll's margin of error. The sampling distribution and its standard error are the gateway to all of statistical inference.</div>
</div>

<div style="text-align:center;color:#8b94b3;font-size:12px;margin-top:16px">Statistics, Data Science and AI: A Visual Handbook · © 2026 John Fisher</div>